uniffi_build/Cargo.toml

```diff
@@ -12,7 +12,7 @@ keywords = ["ffi", "bindgen"]
 
 [dependencies]
 anyhow = "1"
-uniffi_bindgen = { path = "../uniffi_bindgen", optional = true, version = "0.5.0" }
+uniffi_bindgen = { path = "../uniffi_bindgen", optional = true, version = "^0.6.0"}
```

Comment:
nit: In my idealized imagination, we'd just have "0.6.0", without the caret, since I'm not convinced that we want to complicate our (theoretical) testing matrix in this way.  If that's easy to do with `cargo release`, that'd be nice.  That said, if it's not, (maybe it's part of the fix to sunng87/cargo-release?), it's probably not worth worrying about.

Reply:
Yeah, this was the default behavior of `cargo release`, I couldn't see a way to change it, and it didn't seem like enough of a problem relative to the value of not having to make all these changes by hand.

Reply:
Aha, looks like using an explicit `=` requirement rather than an implicit `^` will do the right thing, and is more in line with our expectations anyway.

Reply:
Sounds good.